docs/js/doc-page.js

```diff
@@ -30,7 +30,11 @@ function relativeToAbsUrl(slug) {
 }
 
 function getMainDocsUrl(slug) {
-  return relativeToAbsUrl(slug.match(/\/docs\/[\d\.(latest)]*/)[0]);
+  // All doc pages slugs follow the pattern /docs/{VERSION}/{PATH}
+  // This regex extracts the /docs/{VERSION}/ part to find the current page's main docs slug
+  const mainDocsSlug = getPathFromSlug(slug).match(/\/docs\/[\d\.(latest)]*/)
```

Comment:
Let me rephrase: I don't think that regex does what you think it does.

```console
$ node
Welcome to Node.js v14.4.0.
Type ".help" for more information.
> "/docs/()tassel()/foo".match(/\/docs\/[\d\.(latest)]*/)
[
  '/docs/()tassel()',
  index: 0,
  input: '/docs/()tassel()/foo',
  groups: undefined
]
```

Reply:
This is exactly the expected behavior: `()tassel()` is the docs version in the `/docs/()tassel()/foo` URL, so the main docs path would be `getPathFromSlug('/docs/()tassel()')`, which is the first element of the resulting `match` operation (`match(...)[0]`) 😄

Does this make sense? I've tested it manually with all docs versions, but maybe we can create a new PR adding unit tests?

Reply:
So that's not what a glance at the regex suggests.  If we don't care specifically about the string `latest`, it probably shouldn't be confusingly in the regex.  Perhaps the simpler `/\/docs\/[^/]*/`?